### PR TITLE
Add stubs for new endpoints for execution data

### PR DESCRIPTION
**Pull request by illia-malachyn**
This backports b9b26360cebb722ac0c3ea421e04140d1777fc62 (https://github.com/onflow/flow-emulator/pull/639) to the master branch

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -210,3 +210,5 @@ require (
 	nhooyr.io/websocket v1.8.7 // indirect
 	rsc.io/tmplfunc v0.0.3 // indirect
 )
+
+replace github.com/onflow/flow-go v0.33.2-0.20240425031927-4246f1af6031 => github.com/The-K-R-O-K/flow-go v0.33.2-0.20240507090417-9acec65fa54d
```
Comment (illia-malachyn) [inline go.mod:214]: If try to update this or remove, the bunch of errors come up

Comment (peterargue) [reply go.mod:214]: this is OK in the first merge in `flow-emulator`. we'll submit a follow up PR to switch it to a commit in `flow-go` after your PR is merged

**Review by peterargue — COMMENTED** [review]
please also update the version of `github.com/onflow/flow/protobuf/go/flow` to `v0.4.1`

**Comment by illia-malachyn** [comment]
> please also update the version of `github.com/onflow/flow/protobuf/go/flow` to `v0.4.1`

When I do this, the jobs on CI fail; though everything is fine locally.

p.s. sry for force-pushing this again

**Comment by illia-malachyn** [comment]
I synced [PR flow-go](https://github.com/onflow/flow-go/pull/5672) with master and made this PR point to the latest commit in [PR](https://github.com/onflow/flow-go/pull/5672). Now jobs don't fail

**Review by bluesign — APPROVED** [review]

**Pull request merged**